### PR TITLE
Add support for multi-container apps to azure_rm_webapp

### PR DESCRIPTION
**Pull request by kangasta**
##### SUMMARY

Pass-through container name to applications linux version if it starts with one of the allowed values for multi-container application, `COMPOSE|` or `KUBE|`, in order to allow configuring multi-container applications through existing container settings.

##### ISSUE TYPE

- Feature Pull Request

##### COMPONENT NAME

azure_rm_webapp.py

### CODE REVIEW

File: plugins/modules/azure_rm_webapp.py [hunk 1]
```diff
@@ -100,7 +100,7 @@
         suboptions:
             name:
                 description:
-                    - Name of container, for example C(imagename:tag).
+                    - Name of the container, for example C(imagename:tag). To create a multi-container app, the name should be C(COMPOSE|) or C(KUBE|) followed by base64 encoded configuration.
```
Comment (Fred-sun) [inline plugins/modules/azure_rm_webapp.py]: Why use such a naming method? Is this different from normal naming?
```suggestion
                    - Name of the container, for example C(imagename:tag).
                    - To create a multi-container app, the name should be 'COMPOSE|' or 'KUBE|' followed by base64 encoded configuration.
```

Comment (kangasta) [reply plugins/modules/azure_rm_webapp.py]: This is the format that Azure seems to use to differentiate between single and multi-container applications. `az webapp config container show` returns the containers name in this format if webapp is a multi-container application.

Comment (Fred-sun) [reply plugins/modules/azure_rm_webapp.py]: I see! Thank you very much!

File: plugins/modules/azure_rm_webapp.py [hunk 2]
```diff
@@ -234,6 +235,16 @@
...
 
+    - name: Create a multi-container web app
+      azure_rm_webapp:
+        resource_group: myResourceGroup
+        name: myMultiContainerWebapp
+        plan: myAppServicePlan
+        app_settings:
+          testkey: testvalue
+        container_settings:
+          name: "COMPOSE|{{ lookup('file', 'docker-compose.yml') | b64encode }}"
```
Comment (Fred-sun) [inline plugins/modules/azure_rm_webapp.py:246]: COMPOSE and DOCKER should be followed by lowercase。
```suggestion
          name: COMPOSE{{ lookup('file', 'docker-compose.yml') | b64decode }}
```

**Comment by Fred-sun** [comment]
kindly ping!

File: plugins/modules/azure_rm_webapp.py (hunk 2 above)
Comment (kangasta) [reply plugins/modules/azure_rm_webapp.py:246]: To my understanding, Azure webapp API uses base64 encoded docker-compose file content with `COMPOSE|` prefix as the container name when deploying multi-container application. For example: https://docs.microsoft.com/en-us/azure/app-service/tutorial-multi-container-app#update-app-with-new-configuration

**Pull request merged**